web/src/utils/analytics.ts

```diff
@@ -1,29 +1,17 @@
+import { Capacitor } from '@capacitor/core';
+import posthog from 'posthog-js';
 import { Charts, TrackEvent } from 'utils/constants';
 
-type PlausibleEventProps = { readonly [propName: string]: string | number | boolean };
-type PlausibleArguments = [string, { props: PlausibleEventProps }];
-
-// TODO: Consider moving this to its own global file
-declare global {
-  const plausible: {
-    (...arguments_: PlausibleArguments): void; // Renamed 'arguments' to 'arguments_' to avoid conflict with the reserved keyword 'arguments'
-    q?: PlausibleArguments[];
-  };
-
-  interface Window {
-    plausible?: typeof plausible | undefined;
-  }
-}
-
-export default function trackEvent(
-  eventId: TrackEvent,
-  additionalProps: PlausibleEventProps = {}
-): void {
+export function trackEvent(eventName: TrackEvent, additionalProps = {}): void {
   if (import.meta.env.DEV) {
-    console.log("not sending event to plausible because we're not in production");
+    console.log("not sending event to posthog because we're not in production");
     return;
   }
-  window.plausible?.(eventId, { props: additionalProps });
+  if (Capacitor.isNativePlatform()) {
+    console.log('not sending event to posthog in native apps');
+    return;
+  }
```

Comment:
It's doubtful anyone will read the logs inside the iOS and Android logs, we can remove the logging here.